frontend/src/app/main/data/workspace.cljs

```diff
@@ -2005,6 +2014,20 @@
              (dwu/commit-undo-transaction))))))))
 
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Colorpalette
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn save-colorpalete-status
```

Comment:
Don't like very much to have a separated function for this. I think it would be better to modify functions like toggle-layout-flags and remove-layout-flags functions, to check the flags, and store them in local storage when needed.

This would allow to easily extend the behavior to other layout flags. For example, to text palettes, that are not mentioned in the issue, but are very likely to be also needed.